server.js

```diff
@@ -63,36 +82,49 @@ app.use(function (req, res, next) {
   next();
 });
 
+// Redirect root to /docs when in promo mode.
+if (promoMode == 'true'){
+  console.log('Kit running in promo mode');
+  app.get('/', function (req, res) {
+    res.redirect('/docs');
+  });
+}
+
 // routes (found in app/routes.js)
 if (typeof(routes) != "function"){
   console.log(routes.bind);
-  console.log("Warning: the use of bind in routes is deprecated - please check the prototype kit documentation for writing routes.")
+  console.log("Warning: the use of bind in routes is deprecated - please check the prototype kit documentation for writing routes.");
   routes.bind(app);
 } else {
   app.use("/", routes);
 }
 
+// Downloads returns a url to the zip of the latest release on github
+app.get('/prototype-admin/download-latest', function (req, res) {
+  url = utils.getLatestRelease();
+  res.redirect(url);
+});
+
+if (useDocs == true){
```

Comment:
comparison to a boolean should use === otherwise [JavaScript can do weird things](https://developer.mozilla.org/en-US/docs/Glossary/Truthy).